client/settings/general-settings/test/general-settings.test.js

```diff
@@ -71,4 +71,43 @@ describe( 'GeneralSettings', () => {
 			);
 		}
 	);
+
+	it.each( [ [ true ], [ false ] ] )(
+		'displays Test Mode enabled = %s state from data store',
```

Comment:
It's a bit confusing from reading the name of this test case: 

```suggestion
		'displays CheckBox should be checked when initial data store testMode = %s',
```

Reply:
This name is in the same pattern as the other tests in the file, and intentionally left generic as it runs for both true and false, so we cannot state that "checkbox should be checked."

Reply:
That's fair. So I suggest `displays CheckBox controlling test mode when initial data store testMode = %s`. I think, we'd like to test if the checkbox is displayed rather than the test mode itself. Hope that makes sense now.

Reply:
Went with `display of CheckBox when initial Test Mode = %s`